crates/m3/src/gadgets/hash/keccak/test_vector.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2025 Irreducible Inc.
+
+#![cfg(test)]
+
+const N: usize = 2;
+
+/// N test vectors for keccak-f[1600].
+///
+/// [in, out] where in & out is a state matrix (5x5=25) in a row-major order.
+pub static TEST_VECTOR: [[[u64; 25]; 2]; N] = [
+	[
+		[
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+			0x0000000000000000,
+		],
+		[
+			0xF1258F7940E1DDE7,
+			0x84D5CCF933C0478A,
+			0xD598261EA65AA9EE,
+			0xBD1547306F80494D,
+			0x8B284E056253D057,
+			0xFF97A42D7F8E6FD4,
+			0x90FEE5A0A44647C4,
+			0x8C5BDA0CD6192E76,
+			0xAD30A6F71B19059C,
+			0x30935AB7D08FFC64,
+			0xEB5AA93F2317D635,
+			0xA9A6E6260D712103,
+			0x81A57C16DBCF555F,
+			0x43B831CD0347C826,
+			0x1F22F1A11A5569F,
+			0x5E5635A21D9AE61,
```

Comment:
very nitpicky, but I prefer 0-padding on the left in cases like this?

Reply:
oh yeah I agree! bulls eye